api/v1alpha1/common.go

```diff
@@ -243,6 +243,28 @@ const (
 	DataProcessType OperationType = "DataProcess"
 )
 
+type AffinityPolicy string
```

Comment:
We need add comments for AffinityPolicy:  

```
// AffinityPolicy type defines the strategy for the affinity between Data Operation Pods.
```

Reply:
fixed.